pkg/karmadactl/cmdinit/options/global.go

```diff
@@ -39,4 +39,14 @@ const (
 	KarmadaKubeConfigName = "karmada-apiserver.config"
 	// WaitComponentReadyTimeout wait component ready time
 	WaitComponentReadyTimeout = 120
+
+	// KarmadaConfigSecretSuffix karmada config secret suffix
+	KarmadaConfigSecretSuffix = "-config" //nolint:gosec
+	// KarmadaConfigFieldName the field stores karmada config in karmada config secret
+	KarmadaConfigFieldName = "karmada.config" //nolint:gosec
 )
+
+// KarmadaConfigName returns the name of karmada config secret
+func KarmadaConfigName(component string) string {
+	return component + KarmadaConfigSecretSuffix
+}
```

Comment:
This is shared by both `addon` and `init`, why not put it to package `pkg/karmadactl/util`?